public/api/v1/incompatible-mods.json

```diff
@@ -122,5 +122,22 @@
     "tracking": "UNKNOWN",
     "note": "Compostable items cannot be composted on Quilt",
     "icon": "https://cdn.modrinth.com/data/HJetCzWo/baf9ea57b7784e743aa33441105723205d743351.png"
-  }	 	  
+  },
+  {
+    "ids": ["requiem"],
+    "name": "Requiem",
+    "type": "WORKAROUND",
+    "status": "WONT_FIX",
+    "tracking": "https://github.com/Ladysnake/Requiem/issues/670",
+    "note": "Requires a specific version of Quilt Standard Libraries on 1.19.2: https://modrinth.com/mod/qsl/version/4.0.0-beta.30+0.76.0-1.19.2",
+    "icon": "https://cdn.modrinth.com/data/bYmqMxlQ/41b48c13bc4acd691dccc93b09c97995f0509880_96.webp"
+  },
+  {
+    "ids": ["progressive-archery"],
+    "name": "Progressive Archery",
+    "type": "GAME",
+    "status": "UNKNOWN",
+    "tracking": "UNKNOWN",
+    "icon": "https://cdn.modrinth.com/data/QgHW53j5/590f3d5c405da0ffe21dd6fb12d2360d382531e7.png"
+  }
```

Comment:
Are there any details about the error that you could add in a note? Even just the top-level exception that it prints would be useful.

Reply:
Upon testing, it seems that the mod doesn't break the launcher by itself - it's possibly an incompatibility with another commonly used mod. I'll wait until more crash reports where this mod is involved to show up before making any further changes to the incompatibility listing. If warranted, can be removed altogether until then

Reply:
I'm going to remove the block for Progressive Archery so that the PR can be reviewed and hopefully merged. If issues with Progressive Archery arise in the future, I'll take a closer look and properly detail the problem with the mod in another PR.